src/main/java/ui/components/pickers/AssigneePicker.java

```diff
@@ -0,0 +1,49 @@
+package ui.components.pickers;
+
+import backend.resource.TurboIssue;
+import backend.resource.TurboUser;
+import javafx.application.Platform;
+import javafx.scene.control.ButtonType;
+import javafx.stage.Stage;
+import ui.UI;
+import undo.actions.ChangeAssigneeAction;
+import util.events.ShowAssigneePickerEventHandler;
+
+import java.util.List;
+import java.util.Optional;
+
+public class AssigneePicker {
+
+    private final UI ui;
+    private final Stage stage;
+
+    public AssigneePicker(UI ui, Stage stage) {
+        this.ui = ui;
+        this.stage = stage;
+        ui.registerEvent((ShowAssigneePickerEventHandler) e -> Platform.runLater(() -> showAssigneePicker(e.issue)));
+    }
+
+    private void showAssigneePicker(TurboIssue issue) {
+        List<TurboUser> assigneeList = ui.logic.getRepo(issue.getRepoId()).getUsers();
+        AssigneePickerDialog assigneePickerDialog = new AssigneePickerDialog(stage, issue, assigneeList);
+        Optional<AssigneePickerDialogResponse> assigneeDialogResponse = assigneePickerDialog.showAndWait();
```

Comment:
Think I said the same things for the milestone picker:

- Types are implementation details, so we can leave them out of variable names. `assignees`?
- We don't really need to name the intermediate result, so we can do `new AssigneePickerDialog(...).showAndWait(...)`.

Also, can the dialog response by an inner class? Then we don't have to prefix it.